### PR TITLE
feature: Export more utils

### PR DESCRIPTION
**Pull request by bayotop**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -45,3 +45,4 @@ export {
   transformSource, // Not used here but adds only 222 uncompressed (60 compressed) bytes of code
   UnknownSources,
 } from './utils/entropy_source'
+export { withIframe } from './utils/dom'
```
Comment (Finesse) [inline src/index.ts:48]: Please add a warning comment used with every other undocumented export

**Review by Finesse — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (Finesse) [reply src/index.ts:48]: It already is, my bad

Comment (bayotop) [reply src/index.ts:48]: I added it below the warning comment above, which I'd assume includes this export too. Should I add another one regardless, to be sure?

Comment (Finesse) [reply src/index.ts:48]: I mean this message that you can see in IDE or in `fp.d.ts`:

image

It's added right above the function declaration.

**Review by Finesse — APPROVED** [review]

**Pull request merged**